ui/src/containers/AlertPage.js

```diff
@@ -163,7 +157,6 @@ export const TableHeader = styled.th`
 
 const AlertContent = styled.div`
```

Comment:
I think we can eventually remove the <AlertContent>, the only issue we have today is the table doesn't take the 100% width of it's parents. I will push a fix in TableV2 core-ui.

BTW, Just to clear the code a bit, we can remove SortCaretWrapper, SortIncentive, TableHeader Style.